WordPress/src/main/java/org/wordpress/android/ui/sitecreation/verticals/SiteCreationIntentsFragment.kt

```diff
@@ -0,0 +1,88 @@
+package org.wordpress.android.ui.sitecreation.verticals
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.isInvisible
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.SiteCreationIntentsFragmentBinding
+import org.wordpress.android.ui.utils.UiHelpers
+import org.wordpress.android.util.DisplayUtilsWrapper
+import javax.inject.Inject
+
+/**
+ * Implements the Site Intent Question UI
+ */
+@Suppress("TooManyFunctions")
```

Comment:
Not suggesting a change here. Just curious, are we actually hitting this limit, or is this here because we anticipate exceeding it in future PRs? (wondering, since, this seems small so far, so maybe the threshold needs adjusting)? :thinking:

Reply:
@mkevins Thank you for spotting this, I think it's safe to remove it, I'll try it out.
Initially I did a bit more UI handling in the PR that I ended up removing to keep the PR under the `300` line changes guideline (it's still in the commit history though).
At that point I was exactly at the limit, but I removed at least a couple of functions.

Reply:
✅ Removed since it's safe without it